src/icon-manifest/supportedExtensions.ts

```diff
@@ -1358,6 +1358,12 @@ export const extensions: IFileCollection = {
       extensions: ['infopathxml', 'xsn', 'xsf', 'xtp2'],
       format: FileFormat.svg,
     },
+    {
+      icon: 'ink',
+      extensions: ['ink', 'ink2'],
```

Comment:
Provided that you´re leveraging `languageId` support you don´t need the extensions. I´ve looked into the [ink vscode extension](https://github.com/sequitur/ink-vscode/blob/master/package.json#L37-L50) and it already support those extensions.

Reply:
Ahh, I see - thank you for explaining that!